mllib-local/src/main/scala/org/apache/spark/ml/linalg/Vectors.scala

```diff
@@ -537,6 +537,22 @@ class DenseVector @Since("2.0.0") ( @Since("2.0.0") val values: Array[Double]) e
       maxIdx
     }
   }
+  
+  /**
+ * Function that returns the result of multiply this DenseVector by another DenseVector.
+ *
+ * @param other The second DenseVector
+ * @return The resulting Double value
+ */
+  def dot(other: DenseVector): Double = {
+    require(this.size == other.size, "DenseVectors dimmensions must be the same." +
```

Comment:
Typo in dimmensions, and the indentation is off below. I don't think we should add this since it's just a call to an existing method.